public/dashboard.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <p id="welcome-text">welcome to your diary</p>
+      <button id="btnLogout">Logout</button>
+    </header>
+
+    <section class="create-entry">
+      <form id="formCreate">
```

Comment:
there is no method here, or its not necessary?

Reply:
yea well handle it using javascript

Reply:
we'll*

Reply:
It is handled in javascript

Reply:
Dashboard.html -> app.js
index.html -> login.js
register.html -> register.js
entry.html -> singleEntry.js

Reply:
I can leave comments for method calls should it better guide you to where you need to look.

Reply:
Alright, Got it.

Don't add too many comments though because some things are obvious, I just wanted to check if we are on the same page.